tests/_helpers.py

```diff
@@ -368,3 +368,406 @@ def _import_rpy2_mgcv() -> bool:
 gamr_skip = pytest.mark.skipif(
     _import_rpy2_mgcv(), reason="Cannot import `rpy2` or R's `mgcv` package."
 )
+
+
+def test_matrix_1() -> np.ndarray:
+    """
+    Parameters
+    ----------
+
+    Returns
+    -------
+    :class:`numpy.ndarray`
+        Row-normalized transition matrix. This matrix is
+        - connected
+        - irreducible
+        - not reversible
+    """
+
+    p = np.array(
+        [
+            # 0.   1.   2.   3.   4.   5.   6.   7.   8.   9.   10.  11.
+            [0.0, 0.8, 0.2, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],  # 0
+            [0.2, 0.0, 0.6, 0.2, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],  # 1
+            [0.6, 0.2, 0.0, 0.2, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],  # 2
+            [0.0, 0.05, 0.05, 0.0, 0.6, 0.3, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],  # 3
+            [0.0, 0.0, 0.0, 0.25, 0.0, 0.25, 0.4, 0.0, 0.0, 0.1, 0.0, 0.0],  # 4
+            [0.0, 0.0, 0.0, 0.25, 0.25, 0.0, 0.1, 0.0, 0.0, 0.4, 0.0, 0.0],  # 5
+            [0.0, 0.0, 0.0, 0.0, 0.05, 0.05, 0.0, 0.7, 0.2, 0.0, 0.0, 0.0],  # 6
+            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.2, 0.0, 0.8, 0.0, 0.0, 0.0],  # 7
+            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.8, 0.2, 0.0, 0.0, 0.0, 0.0],  # 8
+            [0.0, 0.0, 0.0, 0.0, 0.05, 0.05, 0.0, 0.0, 0.0, 0.0, 0.7, 0.2],  # 9
+            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.2, 0.0, 0.8],  # 10
+            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.8, 0.2, 0.0],  # 11
+        ]
+    )
+
+    return p
+
+
+def test_matrix_2() -> np.ndarray:
+    """
+    Parameters
+    ----------
+
+    Returns
+    -------
+    :class:`numpy.ndarray`
+        Row-normalized transition matrix. This matrix is
+        - connected
+        - not irreducible (1 recurrent, 1 transient class)
+        - not reversible
+    """
+
+    p = np.array(
+        [
```

Comment:
Could you please reformat the array back to a block structure? It's really hard to read - you should you:
```python
# fmt: off
<the array>
# fmt: on
```
see [here](https://github.com/psf/black#the-black-code-style).

Reply:
done.